### ISSUE TITLE
bug: `path_exists()` does not respect `set working-directory`

### ISSUE THREAD

**Issue by nyurik** [casey/just#2292]
`path_exists()` function computes relative paths relative to the justfile location, not the working directory.

This one is probably tricky because the order of commands might be important:

```just
with_foo1 := if path_exists("foo") != "true" { 'yes' } else { 'no' }
set working-directory := 'bar'
with_foo2 := if path_exists("foo") != "true" { 'yes' } else { 'no' }
```

**Comment by casey** [comment]
Great find! The order of statements doesn't matter, everything in a justfile works in an order-independent fashion. This was a problem in five functions. Fixed in #2294.

**Comment by nyurik** [comment]
:rocket: :rocket: :rocket:  Insanely fast turnaround!!! Thank you :)
